tests/e2e_tests/photon_count_e2e.py

```diff
@@ -0,0 +1,178 @@
+# Photon Counting E2E Test Code
+
+import argparse
+import os
+import pytest
+import numpy as np
+import astropy.time as time
+from astropy.io import fits
+
+import corgidrp
+import corgidrp.data as data
+import corgidrp.mocks as mocks
+import corgidrp.walker as walker
+import corgidrp.caldb as caldb
+import corgidrp.detector as detector
+
+@pytest.mark.e2e
+def test_expected_results_e2e(tvacdata_dir, file_dir):
```

Comment:
The arguments need to be named `tvacdata_path, e2eoutput_path` exactly for the pytest framework to run correctly. Can you rename these two argument variables?